src/toggle/types.ts

```diff
@@ -0,0 +1,36 @@
+export type ToggleType = "checkbox" | "radio" | "yes" | "no";
+export type ToggleStyleType = "default" | "no-border";
+
+export interface ToggleProps
+    extends React.AriaAttributes,
+        React.DOMAttributes<HTMLInputElement> {
+    /**
+     * The type of toggle. Values: "checkbox" | "radio" | "yes" | "no"
+     *
+     * "checkbox" - allows deselection
+     *
+     * "radio", "yes", "no" - does not allow deselection. However they
+     * will determine the indicator to be displayed
+     */
+    type?: ToggleType | undefined;
+    /** Specifies if the indicator icon should be displayed */
+    indicator?: boolean | undefined;
+    /** Indicates if the element is checked */
+    checked?: boolean | undefined;
+    /** The style type of the Toggle. Values: "default" | "no-border" */
+    styleType?: ToggleStyleType | undefined;
+    /** The Toggle label */
+    children: string;
+    /** The description label displayed below the main label */
```

Comment:
may need to comment out this line with `//` as vscode is associating it with the disabled prop